Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,31 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+
+            if (array == null)
+            {
+                return;
+            }
+
+            var flag = true;
+            var temp = 0;
+            var numLength = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'numLength'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887669)